.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
+- docker
+sudo: required
 node_js:
 - '4.2'
 script:
 - npm test
+- make docker-build
+deploy:
+  provider: script
+  script: scripts/deploy.sh
```

Comment:
Good points. I was neglecting the fact that there is already a travis build for this repository. Yes, I agree that the docker build & push should come from travis, and not happen on Docker Hub's build farm.